src/main/java/com/google/api/generator/gapic/protoparser/Parser.java

```diff
@@ -787,9 +802,22 @@ static boolean parseIsPaged(
     // This should technically handle the absence of either of these fields (aip.dev/158), but we
     // gate on their collective presence to ensure the generated surface is backawrds-compatible
     // with monolith-gnerated libraries.
-    return inputMessage.fieldMap().containsKey("page_size")
-        && inputMessage.fieldMap().containsKey("page_token")
-        && outputMessage.fieldMap().containsKey("next_page_token");
+    String pagedFieldName = null;
```

Comment:
As per the comment thread above, can we remove this null-assignment?

Reply:
This is a legit null assignment, as there is no else block. I.e. if there are not signs of field being paginated, `null` is a legit value, representing non-paginated case.